neurophotonics/pipeline/demix.py

```diff
@@ -77,21 +81,22 @@ class Demix(dj.Computed):
     -> IlluminationCycle
     -> Sample
     ---
-    selection         : longblob  # selected cells
-    mix_norm          : longblob  # cell's mixing vector norm
-    demix_norm        : longblob  # cell's demixing vector norm
-    bias_norm         : longblob  # cell's bias vector norm
-    trans_bias_norm   : longblob  # don't use. Saved just in case of wrong axis choice
-    total_power  : float # (uW) average
-    emitter_power : float # (uW) power when on
+    selection                 : longblob  # selected cells
+    mix_norm                  : longblob  # cell's mixing vector norm
+    demix_norm                : longblob  # cell's demixing vector norm
+    bias_norm                 : longblob  # cell's bias vector norm
+    trans_bias_norm           : longblob  # don't use. Saved just in case of wrong axis choice
+    total_power               : float # (uW) average
+    emitter_power             : float # (uW) power when on
+    detector_quantum_efficiency=Null : float
```

Comment:
it should probably not be nulllable. 
```suggestion
    detector_quantum_efficiency=1.0: float
```